backends/bmv2/jsonconverter.cpp

```diff
@@ -1363,15 +1521,34 @@ JsonConverter::convertTable(const CFG::TableNode* node,
     cstring name = extVisibleName(table);
     result->emplace("name", name);
     result->emplace("id", nextId("tables"));
-    cstring table_match_type = "exact";
+    cstring table_match_type = corelib.exactMatch.name;
     auto key = table->getKey();
     auto tkey = mkArrayField(result, "key");
     conv->simpleExpressionsOnly = true;
 
     if (key != nullptr) {
         for (auto ke : *key->keyElements) {
-            auto mt = refMap->getDeclaration(ke->matchType->path, true)->to<IR::Declaration_ID>();
-            BUG_CHECK(mt != nullptr, "%1%: could not find declaration", ke->matchType);
+            auto match_type = getKeyMatchType(ke);
+            if (match_type == v1model.selectorMatchType.name)
+                    continue;
+            // Decreasing order of precedence:
```

Comment:
Updated comment. It is actually what BMv2 expects (which might be the same as v1model spec).